Settings/ItemList/ItemListEvaluator.cs

```diff
@@ -604,7 +605,7 @@ internal static bool EvaluateProperty(LRule itemRule, TrinityItem item, out floa
                             break;
                         }
 
-                        itemValue = item.Attributes.GetAttribute<float>(attribute, modifierId);
+                        itemValue = item.GetAttribute<float>(attribute, modifierId);

```

Comment:
`Attributes` seems to be renamed to `Stats` but this method is still named GetAttribute. Is this intended?